pkg/volume/azure_dd/vhd_util.go

```diff
@@ -56,41 +84,39 @@ func findDiskByLun(lun int, io ioHandler, exe exec.Interface) (string, error) {
 			if len(arr) < 4 {
 				continue
 			}
-			target, err := strconv.Atoi(arr[0])
+			l, err := strconv.Atoi(arr[3])
```

Comment:
Could we make a note of this change and the reason? Or make it a constant so it is easier to identify in case later we might need to change this again?